Attempt to remove CommComm from README (#742)
nodejs/admin: README.md

# Administration Documents for Node.js

> This repository is formal efforts of the Node.js project that are overseen by the [TSC](https://github.com/nodejs/TSC) regarding the less technical aspects of the project.

Reviewer: ```suggestion
This repository is formal efforts of the Node.js project that are overseen by
the [TSC](https://github.com/nodejs/TSC) regarding the less technical aspects of
the project.
```

- The private [Moderation](https://github.com/nodejs/moderation) repo (Collaborators-only access)

## Travel Fund

Each member of the Node.js organization can ask for funding to travel and spread knowledge about and support the JavaScript ecosystem and the Foundation.

The management of Travel Fund for collaborators is managed by the [OpenJS
Foundation Cross-Project-Council](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/project-resources/MEMBER_TRAVEL_FUND.md).

## Collaborator Summit

Every year, Node.js collaborators meet up in one or more Collaborator Summit. If you are interested in joining or helping with the organization, the management of Collaborators Summit is managed by the [OpenJS Foundation Summit team](https://github.com/openjs-foundation/summit).

## Governance and Current Members

The Admin repo is overseen by the [TSC](https://github.com/nodejs/TSC). Any action requiring consensus and voting will abide by the TSC process for that.

### Contacts for assistance
- [@mhdawson](https://github.com/mhdawson) - **Michael Dawson**, TSC Chair
- [@mcollina](https://github.com/mcollina) - **Matteo Collina**, TSC Vice Chair

> - [@joesepi](https://github.com/joesepi) - **Joe Sepi**, CommComm OpenJS Representative

Reviewer 2: @joesepi isn't the CommComm representative - there is no CommComm :P

Reviewer 3: Joe and I are the two Node.js reps on the CPC. Both reps are appointed by TSC. It used to be that one was TSC and one was CommComm.

Neither of us need to be listed here. We don't manage the admin repo. TSC does. So just listing the TSC chair and vice chair seems good enough.

Reviewer: ```suggestion
```

# Administration Documents for Node.js

This repository is formal efforts of the Node.js project that are overseen by the [TSC](https://github.com/nodejs/TSC) regarding the less technical aspects of the project.

It is also used to discuss:

- [Getting Started](https://github.com/nodejs/getting-started)
- The private [Moderation](https://github.com/nodejs/moderation) repo (Collaborators-only access)

## Travel Fund

Each member of the Node.js organization can ask for funding to travel and spread knowledge about and support the JavaScript ecosystem and the Foundation.

The management of Travel Fund for collaborators is managed by the [OpenJS
Foundation Cross-Project-Council](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/project-resources/MEMBER_TRAVEL_FUND.md).

## Collaborator Summit

> Every year, Node.js collaborators meet up in one or more Collaborator Summit. If you are interested in joining or helping with the organization, the management of Collaborators Summit is managed by the [OpenJS Foundation Summit team](https://github.com/openjs-foundation/summit).

Reviewer: Not sure I like this change, I'd prefer the markdown to be wrapped at a reasonable line length (e.g. 80 chars). But granted it's a bit inconsistent with the rest of the file, I'd like to see the whole file wrapped at a given line length, although feel free to ignore as it's a personal preference.

Author: I'm happy to make a follow up change making everything (or nothing) wrap. I tried doing it here but the diff is almost everything.

Reviewer: ```suggestion
Every year, Node.js collaborators meet up in one or more Collaborator
Summit. If you are interested in joining or helping with the
organization, the management of Collaborators Summit is managed by the [OpenJS
Foundation Summit team](https://github.com/openjs-foundation/summit).
```

## Governance and Current Members

The Admin repo is overseen by the [TSC](https://github.com/nodejs/TSC). Any action requiring consensus and voting will abide by the TSC process for that.

### Contacts for assistance
- [@mhdawson](https://github.com/mhdawson) - **Michael Dawson**, TSC Chair

> - [@mcollina](https://github.com/mcollina) - **Matteo Collina**, TSC Vice Chair

Author: Are these the right contacts?

Reviewer 4: yes

### Current account owners:
There will always be two owners of this account - the TSC and CommComm chairpeople. These owners are expected to use their access only to maintain the account and not for access to credientials that they do not otherwise explicitly have access to through [Groups](#1password-groups) within.

- [@bnb](https://github.com/bnb) - **Tierney Cyren**, CommComm Chair
- [@mhdawson](https://github.com/mhdawson) - **Michael Dawson**, TSC Chair
Thanks to 1Password's [open-source program](https://github.com/1Password/1password-teams-open-source), Node.js has been comped a paid version of 1Password, owned by the TSC.

### 1Password Groups

Groups are created in 1Password with a 1:1 mapping of Node.js organization teams, as requested. Groups are asssigned to Vaults, which contain distinct sets of credentials that should only be used by the Groups that are provided access to them. If there is a team in the Node.js organization that would like to have a Group for your Node.js organization team, please PR an addition to the list below and request the current account owners as reviewers on the PR.

> #### Current Groups

Author: I figured it's easier since it is being removed?

Reviewer: The Social Team is still a thing, its repo was archived because it didn't serve much purpose any more, but the team is active with the members listed. I suggest to remove those changes, as those have nothing to do with CommComm.

> ## Node.js Project Calendar

Author: I would need help from @nodejs/tsc regarding who maintains the calendar now?

Reviewer: Let's discuss that in a follow up PR.

## Governance and Current Members

> The Admin repo is overseen by the [TSC](https://github.com/nodejs/TSC). Any action requiring consensus and voting will abide by the TSC process for that.

Reviewer: ```suggestion
The Admin repo is overseen by the [TSC](https://github.com/nodejs/TSC). Any
action requiring consensus and voting will abide by the TSC process for that.
```

## Governance and Current Members

The Admin repo is overseen by the [TSC](https://github.com/nodejs/TSC). Any action requiring consensus and voting will abide by the TSC process for that.

### Contacts for assistance

- [@mhdawson](https://github.com/mhdawson) - **Michael Dawson**, TSC Chair
- [@mcollina](https://github.com/mcollina) - **Matteo Collina**, TSC Vice Chair
- [@joesepi](https://github.com/joesepi) - **Joe Sepi**, CommComm OpenJS Representative

> - [@mmarchini](https://github.com/mmarchini) - **Mary Marchini**, TSC OpenJS Representative

Reviewer: ```suggestion
```

## Node.js 1Password

> Thanks to 1Password's [open-source program](https://github.com/1Password/1password-teams-open-source), Node.js has been comped a paid version of 1Password, managed by the TSC chair person.

Reviewer: ```suggestion
Thanks to 1Password's [open-source program](https://github.com/1Password/1password-teams-open-source),
Node.js has been comped a paid version of 1Password, managed by the TSC chair person.
```

## Node.js 1Password

> Thanks to 1Password's [open-source program](https://github.com/1Password/1password-teams-open-source), Node.js has been comped a paid version of 1Password, owned by the TSC.

Reviewer: ```suggestion
Thanks to 1Password's [open-source program](https://github.com/1Password/1password-teams-open-source), Node.js has been comped a paid version of 1Password, managed by the TSC chair person.
```
